### PR TITLE
Rework cost estimation gas prices

### PR DESCRIPTION
**Pull request by justinkaseman**
TODO:
- [ ] Rework mumbai gas cost estimation
- [ ] Add simulator estimations to `request` command

### CODE REVIEW

File: tasks/Functions-client/request.js [hunk 1]
```diff
@@ -101,7 +101,14 @@ task("functions-request", "Initiates a request from an Functions client contract
     const request = await buildRequest(requestConfig)
 
     // Estimate the cost of the request
-    const { lastBaseFeePerGas, maxPriorityFeePerGas } = await hre.ethers.provider.getFeeData()
+    let gasPriceEstimate
+    const { lastBaseFeePerGas, maxPriorityFeePerGas, gasPrice } = await hre.ethers.provider.getFeeData()
+    if (network.name === "mumbai") {
+      gasPriceEstimate = gasPrice.mul(lastBaseFeePerGas)
```
Comment (bolekk) [inline tasks/Functions-client/request.js:107]: wow, is that what we're supposed to do on Mumbai? Can you add a comment explaining why + a source link if any?

Comment (KuphJr) [reply tasks/Functions-client/request.js:107]: I am also curious as well.  Why does Mumbai require this different estimation?

**Review by KuphJr — APPROVED** [review]
Approving now as it appears to work just fine, but I am still curious for an explanation of the difference between networks

**Review by bolekk — CHANGES_REQUESTED** [review]
This approximation can break any time. Let's not merge it.

**Comment by bolekk** [comment]
We should debug and fix it properly, not use an arbitrary approximation.

**Pull request closed without merging**